source/PlanetLabel.cpp

```diff
@@ -82,12 +82,14 @@ PlanetLabel::PlanetLabel(const Point &position, const StellarObject &object, con
 			if(&other == &object)
 				continue;
 
-			double minDistance = other.Radius() + MIN_DISTANCE;
+			double minDistance = (other.Radius() * zoom) + MIN_DISTANCE;
```

Comment:
```suggestion
			double minDistance = (other.Radius() + MIN_DISTANCE) * zoom;
```

Thoughts? When completely zoomed out this allows labels to be nearer to other stellar objects

Reply:
I'd like to see the difference.

Reply:
Original:

![planet-label-original](https://user-images.githubusercontent.com/85879619/169971696-3f70312e-3261-4c58-901b-4d0046e7a98d.png)

This suggestion:

![planet-label-my-suggestion](https://user-images.githubusercontent.com/85879619/169971802-345fb20f-f3db-4b69-bd75-a150b6982280.png)